Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewRenderer.cs

```diff
@@ -97,10 +97,16 @@ protected virtual void UpdateItemsSource()
 				incc.CollectionChanged += ItemsChanged;
 			}
 
-			ListViewBase.ItemsSource = CollectionViewSource.View;
+			SetItemSourceAndEmptyView();
+		}
+
+		void SetItemSourceAndEmptyView()
```

Comment:
Not calling the rest of the UpdateItemsSource body before the ListViewBase has a size caused some issues with carousel view.  So I separated setting the source and empty view to another method.

Reply:
Now the rest will be called as soon as the CollectionViewSource is set, but will only be passed to the ListViewBase once it's displayed

Reply:
@bmacombe Looks good and great to hear this solves the issue in the repro page!

Just one minor thing: `ListViewBaseHasSize` should probably match the naming format for private fields used in the XF project i.e. _bool _listViewBaseHasSize;_ and declared at the top of the ItemsViewRenderer along with all the other private fields used.

Reply:
oh, yeah it should.  Good catch.